app/models/post.rb

```diff
@@ -2,4 +2,15 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :likes
```

Comment:
- [ ] **[OPTIONAL]** In your migration file the `posts`, `comments`, and `likes` tables should reference `author_id`, and not the `user_id` something like this `t.references :author, foreign_key: { to_table: 'users' }` . This was supposed to be done correctly in the previous milestone. This `belongs_to :user` is supposed to be `belongs_to :author, class_name: 'User'` according to the ERD provided.